test/gtest/ucp/test_ucp_tag.cc

```diff
@@ -689,7 +689,7 @@ UCS_TEST_P(test_ucp_tag_nbx, rndv, "RNDV_THRESH=0")
     test_recv_send(64 * UCS_KBYTE);
 }
 
-UCS_TEST_P(test_ucp_tag_nbx, fallback)
+UCS_TEST_P(test_ucp_tag_nbx, fallback, "ZCOPY_THRESH=inf")
```

Comment:
Why?

Reply:
seems that fallback scenario is not supported for zcopy. (This is also how it works on master today).